### ISSUE TITLE
Create Article: Deploy using Git

### ISSUE THREAD

**Issue by abhisekp** [FreeCodeCamp/wiki#767]
<h1 align="center">Introducing Git Deployment</h1>

Write an article about website deployment using Git. :octocat: 

Ref.: https://coderwall.com/p/xczkaq/ftp-is-so-90-s-let-s-deploy-via-git-instead
<br>
<br>

> ----
<p align="center">Read <a href="https://github.com/FreeCodeCamp/wiki/blob/master/.github/CONTRIBUTING.md"><b>CONTRIBUTING</b></a> Guidelines<br>Chat with us in <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"><img src="http://i.imgur.com/ThSWa6Y.png?2"> <b>FreeCodeCamp/Wiki</b></a></kbd></p>

> ----

**Comment by fulvi0** [comment]
@abhisekp, i'll like to make contribution about this topic, but I have a question about, when you said website could be a static one?

**Comment by abhisekp** [comment]
@fulvi0 :sparkles: thanks for your interest in contributing. 
We have an article about deploying to Github pages here :point_right: [**Using Github Pages for your front end development projects**](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Using-Github-Pages-for-your-front-end-development-projects). 
You may improve the existing article (*either by removing, updating, enhancing or adding*) content or improving the format/layout of the article for better understanding of the article :)

----
**Clarification about this issue**
This issue requests for an article about deploying to a live 3rd party production server using Git.
Please check the reference webpage in https://github.com/FreeCodeCamp/wiki/issues/767#issue-149948837 :point_up:

**Comment by fulvi0** [comment]
Thank you for the information @abhisekp, I already got a [draft](https://github.com/fulvi0/wiki/blob/intro-git-deployment/intro-git-deployment.md), if you can take a look; so I will write the article but focus on deploy a website using Heroku.

**Comment by alayek** [comment]
@fulvi0 it's better that you raise a pull request against our Wiki. This way we can directly review and merge your content.

**Comment by fulvi0** [comment]
@alayek yes, but still i'm working on it, a now I'll make couple of change for the article deploying to live 3rd party production server using Git.

Thanks for the advices.

**Comment by abhisekp** [comment]
@fulvi0 if you're working on it, that's fine. You can file a Pull Request (PR) of the draft and in the PR message just say that you're working on it i.e. **`in progress`** and then just add more and more comments on top of it. 

We'll give it a "`in progress`" label so whenever you're finished, just add a comment in the PR that you're complete and the Wiki Team :shipit: :shipit: :shipit: :shipit: will review the PR. :100:
